.github/workflows/docs.yml

```diff
@@ -154,15 +150,41 @@ jobs:
         uses: nwtgck/actions-netlify@v3.0
         with:
           publish-dir: .
-          production-deploy: ${{ github.ref == 'refs/heads/main' }}
```

Comment:
Also the naming of the job is currently misleading with the two outcomes in the same job.  It took me awhile to figure out what was going on here.

Reply:
>Also the naming of the job is currently misleading with the two outcomes in the same job

Fair, I named it `preview` here because it is strictly dealing with our development documentation. A production deploy here just means that its the `main` branch and our current documentation build -- a "preview" of the next release.